hubspot/plugin.py

```diff
@@ -1002,9 +1013,12 @@ def hubspot_search_contacts(
     # Construct the filters based on the search criteria
     filters = []
     for property_name, value in search_criteria.items():
-        filters.append(
-            {"propertyName": property_name, "operator": "EQ", "value": value}
-        )
+        if value:
+            filters.append(
+                {"propertyName": property_name, "operator": "EQ", "value": value}
+            )
+    if not filters:
```

Comment:
add a comment on why we are doing this check

Reply:
Added comment below this check